frontend/src/routes/(authenticated)/project/create/+page.svelte

```diff
@@ -177,8 +185,9 @@
     <Input
       label={$t('project.create.code')}
       bind:value={$form.code}
-      error={$errors.code}
+      error={$codeErrors}
```

Comment:
I'm guessing from this switch we can't just set `errors.code` ourselves?

Reply:
Honestly, I didn't really even try that myself, because @rmunn had trouble and...I would expect that that wouldn't work 🤷.

Reply:
So, I ended up trying it out quick and yeah, it's a mess.